packages/block-editor/src/hooks/block-bindings.js

```diff
@@ -77,10 +77,10 @@ function BlockBindingsPanelDropdown( { fieldsList, attribute, binding } ) {
 								checked={ key === currentKey }
 							>
 								<DropdownMenuV2.ItemLabel>
-									{ key }
+									{ args?.label }
```

Comment:
Should it fallback to `key` when the label is undefined?

Reply:
Right, that makes perfect sense. I wasn't paying enough attention and I assumed it's all specific to post meta 👍🏻